diff -durN gambatte_src-0.4.1/gambatte_qt/src/framework/framework.pro gambatte_src-0.4.1.orig/gambatte_qt/src/framework/framework.pro
--- gambatte_src-0.4.1/gambatte_qt/src/framework/framework.pro	2009-01-10 23:32:09.000000000 +0100
+++ gambatte_src-0.4.1.orig/gambatte_qt/src/framework/framework.pro	2012-03-17 20:22:44.722324173 +0100
@@ -111,6 +111,10 @@
         framework/fullmodetogglers/xrandr12toggler.h #\
 #        framework/fullmodetogglers/xf86vidmodetoggler.h
     LIBS += -L/usr/X11R6/lib \
+        -Wl,--as-needed \
+        -lX11 \
+        -lXext \
+        -ldl \
         -lXv \
         -lXrandr #\
 #        -lXxf86vm \
