diff -dur gambatte_src-0.4.0.orig/gambatte_sdl/SConstruct gambatte_src-0.4.0/gambatte_sdl/SConstruct
--- gambatte_src-0.4.0.orig/gambatte_sdl/SConstruct	2008-10-28 00:03:47.000000000 +0100
+++ gambatte_src-0.4.0/gambatte_sdl/SConstruct	2008-11-03 16:43:32.000000000 +0100
@@ -26,9 +26,6 @@
 
 conf = env.Configure()
 
-if conf.CheckLib('z'):
-	env.Append(LIBS = ['z'])
-
 conf.Finish()
 
 env.Program('gambatte_sdl', sourceFiles)
diff -dur gambatte_src-0.4.0.orig/libgambatte/SConstruct gambatte_src-0.4.0/libgambatte/SConstruct
--- gambatte_src-0.4.0.orig/libgambatte/SConstruct	2008-10-28 00:03:48.000000000 +0100
+++ gambatte_src-0.4.0/libgambatte/SConstruct	2008-11-03 16:41:38.000000000 +0100
@@ -52,10 +52,8 @@
 
 conf = env.Configure()
 
-if conf.CheckHeader('zlib.h') and conf.CheckLib('z'):
-	sourceFiles.append('src/file/unzip/unzip.c')
-	sourceFiles.append('src/file/unzip/ioapi.c')
-	sourceFiles.append('src/file/file_zip.cpp')
+if conf.CheckLib('z', autoadd=1) and conf.CheckLib('minizip', autoadd=1):
+        sourceFiles.append('src/file/file_zip.cpp')
 else:
 	sourceFiles.append('src/file/file.cpp')
 
diff -dur gambatte_src-0.4.0.orig/libgambatte/src/file/file_zip.cpp gambatte_src-0.4.0/libgambatte/src/file/file_zip.cpp
--- gambatte_src-0.4.0.orig/libgambatte/src/file/file_zip.cpp	2008-10-28 00:03:48.000000000 +0100
+++ gambatte_src-0.4.0/libgambatte/src/file/file_zip.cpp	2008-11-03 16:42:34.000000000 +0100
@@ -22,7 +22,7 @@
 #include <cstring>
 
 namespace zlib {
-#include "unzip/unzip.h"
+#include <minizip/unzip.h>
 }
 
 using namespace std;
