diff -dur gambatte_src-0.4.1.orig/gambatte_qt/src/framework/framework.pro gambatte_src-0.4.1/gambatte_qt/src/framework/framework.pro
--- gambatte_src-0.4.1.orig/gambatte_qt/src/framework/framework.pro	2009-01-10 23:32:09.000000000 +0100
+++ gambatte_src-0.4.1/gambatte_qt/src/framework/framework.pro	2009-01-11 10:40:28.000000000 +0100
@@ -68,7 +68,6 @@
 INCLUDEPATH += framework/SDL_Joystick/include
 INCLUDEPATH += $$COMMONPATH
 DEFINES += HAVE_STDINT_H
-QMAKE_CXXFLAGS += -fno-exceptions -fno-rtti
 
 macx {
 #    CONFIG += x86 ppc
diff -dur gambatte_src-0.4.1.orig/gambatte_sdl/SConstruct gambatte_src-0.4.1/gambatte_sdl/SConstruct
--- gambatte_src-0.4.1.orig/gambatte_sdl/SConstruct	2009-01-10 22:47:57.000000000 +0100
+++ gambatte_src-0.4.1/gambatte_sdl/SConstruct	2009-01-11 10:40:28.000000000 +0100
@@ -1,5 +1,5 @@
-global_cflags = ARGUMENTS.get('CFLAGS', '-Wall -Wextra -O2 -fomit-frame-pointer')
-global_cxxflags = ARGUMENTS.get('CXXFLAGS', global_cflags + ' -fno-exceptions -fno-rtti')
+global_cflags = ARGUMENTS.get('CFLAGS')
+global_cxxflags = ARGUMENTS.get('CXXFLAGS')
 global_defines = ' -DHAVE_STDINT_H'
 
 env = Environment(CPPPATH = ['src', '../libgambatte/include', '../common'],
diff -dur gambatte_src-0.4.1.orig/libgambatte/SConstruct gambatte_src-0.4.1/libgambatte/SConstruct
--- gambatte_src-0.4.1.orig/libgambatte/SConstruct	2009-01-10 22:47:58.000000000 +0100
+++ gambatte_src-0.4.1/libgambatte/SConstruct	2009-01-11 10:40:28.000000000 +0100
@@ -1,5 +1,5 @@
-global_cflags = ARGUMENTS.get('CFLAGS', '-Wall -Wextra -O2 -fomit-frame-pointer')
-global_cxxflags = ARGUMENTS.get('CXXFLAGS', global_cflags + ' -fno-exceptions -fno-rtti')
+global_cflags = ARGUMENTS.get('CFLAGS')
+global_cxxflags = ARGUMENTS.get('CXXFLAGS')
 global_defines = ' -DHAVE_STDINT_H -DCHAR_WIDTH_8'
 
 env = Environment(CPPPATH = ['src', 'include', '../common'],
